Command/CreateSiteCommand.php

```diff
@@ -80,7 +80,24 @@ public function execute(InputInterface $input, OutputInterface $output)
             $values[$name] = $input->getOption($name);
 
             while ($values[$name] == null) {
-                $values[$name] = $helper->ask($output, sprintf('Please define a value for <info>Site.%s</info> : ', $name));
+                /*
+                 * NEXT_MAJOR: remove if statement and use the question helper only (true value of if) when dropping sf < 2.5
+                 */
+                if ($this->getHelperSet()->has('question')) {
+                    $question = new \Symfony\Component\Console\Question\Question(
```

Comment:
but wont that create a problem for backwards compatibility?  before symfony 2.5 it doesn't exist